src/styles/_catalog.scss

```
@@ -1,9 +1,34 @@
 .catalog {
-  display: flex;
+  display: grid;
   flex-wrap: wrap;
 
   max-width: 944px;
   margin: 0 auto;
+  gap: 48px;
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, 1fr) ;
+    }
```

It will be better if you fix it everywhere
```suggestion
      grid-template-columns: repeat(4, 1fr) ;
```